src/gobupload/storage/update_table.py

```diff
@@ -87,35 +89,47 @@ def _select_expressions(self):
 
         :return:
         """
+        # example = {
+        #     '_version': '',
+        #     '_application': '',
+        #     '_source_id': '',
+        #     '_source': '',
+        #     '_expiration_date': '',
+        #     'id': '',                   # concat: src_id . src_volgnummer . src_source . bronwaarde
+        #     'src_source': '',
+        #     'src_id': '',
+        #     'src_volgnummer': '',
+        #     'derivation': '',
+        #     'dst_source': '',
+        #     'dst_id': '',
+        #     'dst_volgnummer': '',
+        #     'bronwaarde': '',
+        #     'begin_geldigheid': '',
+        #     'eind_geldigheid': '',
+        # }
+
```

Comment:
Why not put the example in the method comment. But also: what is the purpose of an empty example?

Reply:
Removed it. Was more of a help during development for which columns to return, but this is easy to read in the code now.